dts/arm/ti/msp432p4xx.dtsi

```diff
@@ -35,6 +38,13 @@
 			clocks = <&sysclk>;
 			status = "disabled";
 		};
+		gpioa: gpio@40005000 {
+			compatible = "ti,msp432p4xx-gpio";
+			reg = <0x40005000 0x1000>;
```

Comment:
Does the GPIO driver exist for this platform? I couldn't find one, where did you get this compatible from?
you might need to add something around https://github.com/zephyrproject-rtos/hal_ti/blob/master/simplelink/source/ti/devices/msp432p4xx/driverlib/gpio.c

Reply:
Thanks for your inputs vaishnav, I have started writing a GPIO driver based on your suggestions.